projects/js-packages/components/components/automattic-byline-logo/index.jsx

```diff
@@ -0,0 +1,42 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import classnames from 'classnames';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * AutomatticBylineLogo component definition.
+ *
+ * @param {object} props - Component properties.
+ * @param {string} props.title - Title for SVG.
+ * @param {number} props.height - Height for SVG.
+ * @param {number} props.className - Additional className for the a wrapper, default only: `jp-automattic-byline-logo`.
+ *
+ * @returns {React.Component} AutomatticBylineLogo component.
+ */
+export default function AutomatticBylineLogo( {
+	title = __( 'An Automattic Airline' ),
```

Comment:
You'll want to add the `jetpack` textdomain here. 

```suggestion
	title = __( 'An Automattic Airline', 'jetpack' ),
```

Maybe that's a good opportunity to add the `plugin:@wordpress/eslint-plugin/i18n` plugin to our eslint config for Jetpack components?
https://github.com/Automattic/jetpack/blob/3128bbedf42abc396d0070cbe1e2e08f6d982461/projects/js-packages/components/.eslintrc.cjs#L4


*******

Somewhat related, as we start using those packages in other plugins, we'll have to handle updating those textdomains as we release. Tracking this in #20320.

Reply:
>You'll want to add the jetpack textdomain here.

Ops thanks for catching it.

>Maybe that's a good opportunity to add the plugin:@wordpress/eslint-plugin/i18n plugin to our eslint config for Jetpack components?

Sure. will do.

Reply:
Hi @jeherve added `plugin:@wordpress/eslint-plugin/i18n` in cf9f213. Could you please double check if it works as expected? Thanks 😄

Reply:
It does, thank you!

<img width="1019" alt="Screenshot 2021-07-12 at 10 36 51" src="https://user-images.githubusercontent.com/426388/125256850-21680300-e2fd-11eb-912e-02416d67cf72.png">